src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -928,15 +920,16 @@ namespace winrt::TerminalApp::implementation
     void TerminalPage::_MoveFocus(const Direction& direction)
     {
         const auto focusedTabIndex = _GetFocusedTabIndex();
-        auto focusedTab = winrt::get_self<implementation::Tab>(_tabs.GetAt(focusedTabIndex));
+        winrt::com_ptr<Tab> focusedTab;
```

Comment:
I stylistically hate that we have to do this like this, but I understand that it's necessary -____-

Reply:
It pains me as well to have to throw these clunky lines pretty much everywhere in this file 😭